src/Workspaces/CSharp/Portable/AddImport/CSharpAddImportService.cs

```diff
@@ -0,0 +1,194 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using Microsoft.CodeAnalysis.AddImport;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.AddImport
+{
+    [ExportLanguageService(typeof(IAddImportService), LanguageNames.CSharp), Shared]
+    internal class CSharpAddImportService : IAddImportService
```

Comment:
Do all the use cases of this service happen in C# land? I'd feel better if the VB one existed, even if it's less useful.